src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Map;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    int size;
+    Node<K, V>[] table;
```

Comment:
The `table` array is declared but not initialized. This will cause a `NullPointerException` when trying to access it. You should initialize the `table` array with the initial capacity, e.g., `table = new Node[capacity];`.